src/test/java/org/opentripplanner/middleware/persistence/TripHistoryPersistenceTest.java

```diff
@@ -27,66 +28,77 @@
  * {@link org.opentripplanner.middleware.controllers.api.ApiController} are not themselves tested here.
  */
 public class TripHistoryPersistenceTest extends OtpMiddlewareTest {
+    private static final int LIMIT = 3;
+    private static final String USER_ID = "123456";
     private static final String TEST_EMAIL = "john.doe@example.com";
+    private static final String TRIP_REQUEST_DATE_CREATED_FIELD_NAME = "dateCreated";
+    private static final String TRIP_REQUEST_USER_ID_FIELD_NAME = "userId";
+
+    private static OtpUser otpUser = null;
+    private static TripRequest tripRequest = null;
+    private static TripSummary tripSummary = null;
+    private static TripSummary tripSummaryWithError = null;
+    private static List<TripRequest> tripRequests = null;
+
+    @BeforeAll
+    public static void setup() throws IOException {
+        otpUser = createUser(TEST_EMAIL);
+        tripRequest = createTripRequest(USER_ID);
+        tripSummary = createTripSummary();
+        tripSummaryWithError = createTripSummaryWithError();
+        tripRequests = createTripRequests(LIMIT, otpUser.id);
+    }
 
-    OtpUser user = null;
-    TripRequest tripRequest = null;
-    TripSummary tripSummary = null;
-    List<TripRequest> tripRequests = null;
+    @AfterAll
+    public static void tearDown() {
+        // Note: otpUser.delete() cannot be used because no Auth0 user is created. The method fails on attempting to
+        // delete the random value assigned to the otpUser on initialization. This in turn prevents the otpUser from
+        // being delete from Mongo.
```

Comment:
Again, this just seems like an opportunity to overload the delete method, rather than duplicate a bunch of code.

Reply:
Updated.